### DISCUSSION TITLE
Webhook notification for automation?

### DISCUSSION THREAD

**Discussion by ahgraber** [ArchiveBox/ArchiveBox#1597] [Q&A] [answered]
I'd like to have ArchiveBox archive a link and then send a (configurable) webhook notification so I can automate some downstream processes as each page is archived.  Is this possible?  If so, could you provide some guidance/example on how?  Thank you!

**Comment by pirate** [comment, marked as answer]
Webhook support was added in this PR https://github.com/ArchiveBox/ArchiveBox/pull/1418 there are some screenshots and info there.

It's available in the [new BETA `v0.8.x` releases])(https://github.com/ArchiveBox/ArchiveBox/releases/tag/v0.8.5-rc), but is not yet out in the stable master release.
